crates/serverless_downloader/src/lib.rs

```diff
@@ -20,11 +20,22 @@ pub fn get_bucket() -> Result<Bucket> {
         None,
     )?;
 
-    Ok(Bucket::new(
-        &bucket_name,
-        bucket_region.parse()?,
-        credentials,
-    )?)
+    let endpoint = env::var("S3_ENDPOINT");
+    let bucket = if endpoint.is_ok() {
+        Bucket::new(
+            &bucket_name,
+            Region::Custom {
+                region: bucket_region.parse()?,
+                endpoint: endpoint.unwrap(),
+            },
+            credentials,
+        )?
+        .with_path_style();
```

Comment:
@QuiiBz Do you think its fine that we set the path-style option implicitly if a specific endpoint url is provided or should I change it to have its own ENV var?

Reply:
We can keep it as-is for now